### PR TITLE
Add obs_tbl function to get OBSERVER tables

### PR DESCRIPTION
**Pull request by david-macmahon**

### CODE REVIEW

**Comment by PerezHz** [comment]
Hey @david-macmahon! Many thanks for this contribution 😄! Is there anything else that you would like to add here, or is it ready for review?

**Comment by david-macmahon** [comment]
I think it's pretty much ready for review.  I thought about adding an `Enum` for `Quantities`.  I think it would make a nice addition, but I didn't want to overly complicate things.  Just getting the basic functionality in place seemed like a good start.  Bells and whistles can always be added later.  I tried to follow the `vec_tbl` approach quite closely, but I made all the numerical options be `Real` instead of `T` because I think using `T` on multiple options means that they all have to be passed as the same type and I think it's better to be able to pass `Int` for some and `Float64` or others.

**Comment by PerezHz** [comment]
> I think it's pretty much ready for review. I thought about adding an Enum for Quantities.

I think we can leave it as it is now; I think it's a pretty good start! Unit tests help a lot, thanks for adding those. As you say, we can improve things down the road.

>  I tried to follow the vec_tbl approach quite closely, but I made all the numerical options be Real instead of T because I think using T on multiple options means that they all have to be passed as the same type and I think it's better to be able to pass Int for some and Float64 or others.

As long as we don't enforce some kind of consistency between the types of the `Real` arguments I don't see a need to add type parameters into the mix.

Besides bumping the patch version, I think it's LGTM and am in favor of merging; I'd just wait for @LuEdRaMo's thumbs up, in case he has more comments on this.

File: src/obstbl.jl [hunk 1]
```diff
@@ -0,0 +1,122 @@
+@doc raw"""
```
Comment (PerezHz) [inline src/obstbl.jl:1]: Docstrings are really helpful, many thanks for adding them!

File: test/runtests.jl [hunk 2]
```diff
@@ -123,4 +123,30 @@ end
     @test apophis_1["signature"] == apophis_2["signature"]
     @test all(map((x, y) -> x == y[1:end-1], apophis_1["data"], apophis_2["data"]))
     @test apophis_1["count"] == apophis_2["count"]
-end
\ No newline at end of file
+end
+
+@testset "Observer table generation: obs_tbl" begin
```
Comment (PerezHz) [inline test/runtests.jl:128]: Thanks for adding this unit test 👍

**Review by PerezHz — APPROVED** [review]

**Comment by LuEdRaMo** [comment]
Thanks @david-macmahon for this contribution! I am in favour of merging after bumping the patch version.

**Comment by david-macmahon** [comment]
I bumped the patch version and I think the PR picked it up automatically, but please let me know if you don't see it.  Thanks for making this package in the first place!

**Comment by PerezHz** [comment]
Got the green lights, so we are GO for merging!

**Comment by PerezHz** [comment]
> I bumped the patch version and I think the PR picked it up automatically, but please let me know if you don't see it.

PR is merged and JuliaRegistrator is looking good so far so we should have a new version in a couple of minutes 👍 

> Thanks for making this package in the first place!

Hey, really glad it's helping the community!

**Pull request merged**